web/components/manalinks/create-links-button.tsx

```diff
@@ -72,6 +73,33 @@ function CreateManalinkForm(props: {
     message: '',
   })
 
+  const expireOptions = {
```

Comment:
Hm, it seems kind of weird to have expireOptions just mapping strings to themselves. Some designs I would prefer

1. Use a type for this: `type ExpiresIn = 'day' | 'week' | 'month' | 'never'`
2. Map
   ```
const EXPIRE_DURATIONS = {
  day: '1 day'
  ...
}
   ```
and then use the map to create the option values in the JSX in place of`<option value={expireOptions.never}>Never</option>`
3. Both 1 & 2!